client/src/partials/templates/messages.tmpl.html

```diff
@@ -0,0 +1,23 @@
+<!-- displays "This value is too small for this field." -->
+<p ng-message="min">This value is too small for this field.</p>
```

Comment:
This is a great separation of concerns. These messages should eventually use translation keys.